contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -169,25 +165,25 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
     }
 
     bool success;
-    try IReceiver(receiverAddress).onReport(workflowId, workflowOwner, rawReport[REPORT_METADATA_LENGTH:]) {
+    try IReceiver(receiverAddress).onReport(rawReport[41:105], rawReport[REPORT_METADATA_LENGTH:]) {
       success = true;
     } catch {
       // Do nothing, success is already false
     }
 
-    s_reports[reportId] = DeliveryStatus(msg.sender, success);
+    s_reports[id] = DeliveryStatus(msg.sender, success);
 
-    emit ReportProcessed(receiverAddress, workflowOwner, workflowExecutionId, success);
+    emit ReportProcessed(receiverAddress, workflowExecutionId, success);
   }
 
-  function _reportId(address receiver, bytes32 workflowExecutionId) internal pure returns (bytes32) {
+  function _reportId(address receiver, bytes32 workflowExecutionId, bytes2 reportId) internal pure returns (bytes32) {
     // TODO: gas savings: could we just use a bytes key and avoid another keccak256 call
-    return keccak256(bytes.concat(bytes20(uint160(receiver)), workflowExecutionId));
+    return keccak256(bytes.concat(bytes20(uint160(receiver)), workflowExecutionId, reportId));
   }
 
   // get transmitter of a given report or 0x0 if it wasn't transmitted yet
-  function getTransmitter(address receiver, bytes32 workflowExecutionId) external view returns (address) {
-    bytes32 reportId = _reportId(receiver, workflowExecutionId);
+  function getTransmitter(address receiver, bytes32 workflowExecutionId, bytes2 reportName) external view returns (address) {
```

Comment:
reportName -> reportId

Reply:
To distinguish from actual IDs, how about we name this `reportType`?

Reply:
How about `transmissionId`? This would align with `transmitter` and the language we use overall.

Reply:
settled on ID